### DISCUSSION TITLE
Why Zen Mode was removed in 2023.8.1

### DISCUSSION THREAD

**Discussion by sky96111** [hoppscotch/hoppscotch#3386] [Q&A] [answered]
I found zen mode was disappeared in 2023.8.1 , so I search for release note and found this PR https://github.com/hoppscotch/hoppscotch/pull/3337
Which refered to "HFE-137", while I can not found any info about this keyword.

I really enjoy the compact UI that comes with zen mode. I want to know why zen mode was removed.
Thanks.

**Comment by liyasthomas** [comment]
Zen mode often caused confusion among users. Also, we've introduced a new and improved CMD + K search and spotlight feature in the header bar, hence decided not to hide the header anymore. If you prefer a compact UI, you can still collapse the primary navigation bar and secondary sidebar. We removed the Zen mode support for hiding the header bar only.

**Reply by sky96111** [reply]
I agree with zen mode sometimes confused users, while I think it can be better to make description more clear rether than remove it.

Actually I don't think the sidebar is a major factor in taking up space but topbar.
- Search box can be trigged by "CTRL+K"
- Invitees are hardly ever used in individual use
- Workspace same
- The login function only needs to be used once

And the extra space in the top bar allows for more data to be displayed, especially JSON

I appeal to add Zen Mode back

**Comment by sky96111** [comment, marked as answer]
Finally I turned to AdGuard to block entire header, and it works like a charm.
Here is AdGuard's syntax
````
hoppscotch.io###app > div > div.flex.w-screen.h-screen > div.splitpanes.splitpanes--horizontal.no-splitter:first-child > div.splitpanes__pane:first-child > div > header.flex.items-center.justify-between.flex-1.flex-shrink-0.px-2.py-2.space-x-2.overflow-x-auto.overflow-y-hidden
````
